tools/bisection/Dockerfile.perf-matrix

```
@@ -60,28 +60,33 @@ WORKDIR /repo
+       074a2219c90feaf962d0d24f034514033ece5f99 \
+    && [ "$(git -C /tmp/libpinyin-src rev-parse FETCH_HEAD)" = \
+       "074a2219c90feaf962d0d24f034514033ece5f99" ] \
+    && git -C /tmp/libpinyin-src checkout --quiet --detach FETCH_HEAD
 
 # ── Cell A: libpinyin + Tkrzw ────────────────────────────────────────
 
-RUN mkdir -p /tmp/build-A && tar xzf /tmp/libpinyin.tar.gz -C /tmp/build-A \
-    && cp target/model20/extracted/* /tmp/build-A/libpinyin-2.11.91/data/ \
-    && cd /tmp/build-A/libpinyin-2.11.91 \
-    && autoreconf --force --install \
+RUN cp -a /tmp/libpinyin-src /tmp/build-A-src \
```

Hadolint (2.15.1)

[warning] Use WORKDIR to switch to a directory

(DL3003)